data/battle_anim_scripts.s

```diff
@@ -1523,17 +1523,17 @@ Move_VINE_WHIP:
 Move_DRILL_PECK:
 	loadspritegfx ANIM_TAG_IMPACT
 	loadspritegfx ANIM_TAG_WHIRLWIND_LINES
-	createsprite gUnknown_08592CD8, ANIM_ATTACKER, 2, 0
+	createsprite gInclineMonSpriteTemplate, ANIM_ATTACKER, 2, 0
```

Comment:
All uses of `Incline` should be changed to `Bow`, `BowDown`, or `BowHead`.  `Incline` means the opposite of what the sprite looks like it's doing during skull bash, etc.